.github/workflows/build.yaml

```diff
@@ -52,11 +52,14 @@ jobs:
                   queries: +security-and-quality
 
             - name: Bootstrap
-              run: scripts/build/gn_bootstrap.sh
+              run: |
+                  eval `dmalloc -b -l DMALLOC_LOG -i 1 high`;
```

Comment:
Can this be set at runtime based on chip_config_memory_debug_checks?

Reply:
Setting it in code is too late. I suppose it could be in `scripts/tests/gn_tests.sh`.

Reply:
script seems better, the concern is that it'll be harder to repro CI failures locally if we have a lot of configurations here.